hibernate-jpa/build.gradle

```diff
@@ -8,8 +8,12 @@ dependencies {
     api(libs.managed.hibernate.core)
     api(libs.managed.jakarta.transaction.api)
     api(mnValidation.micronaut.validation)
-    compileOnly(mnData.micronaut.data.tx.hibernate)
-    testImplementation(mnData.micronaut.data.tx.hibernate)
+    compileOnly(mnData.micronaut.data.tx.hibernate) {
+        exclude(group: "org.hibernate.orm")
```

Comment:
I think it makes sense to exclude hibernate deps to transition from micronaut-data since this is main place where hibernate deps are declared.